python/tvm/target/datatype.py

```diff
@@ -73,7 +73,12 @@ def get_type_registered(type_code):
     return tvm.runtime._ffi_api._datatype_get_type_registered(type_code)
 
 
-def register_op(lower_func, op_name, target, type_name, src_type_name=None):
+def register_op(lower_func,
+                op_name,
+                target,
+                type_name,
+                src_type_name=None,
+                intrinsic_name=None):
     """Register an external function which computes the given op.
 
     Currently, this will only work with Casts and binary expressions
```

Comment:
Hm, this docstring probably needs updating, it doesn't even make sense to me anymore...